Source/Schema.NET/JsonLdObject.cs

```diff
@@ -43,6 +43,32 @@ public class JsonLdObject
         /// a node. In JSON-LD, a node is identified using the @id keyword:
         /// </summary>
         [DataMember(Name = "@id", Order = 2)]
-        public virtual Uri Id { get; set; }
+        public virtual Uri Id { get; set; }
+
+        /// <inheritdoc/>
+        public bool Equals(JsonLdObject other)
+        {
+            if (other is null)
+            {
+                return false;
+            }
+
+            if (ReferenceEquals(this, other))

```

Comment:
In theory StyleCop would require `this.` before `ReferenceEquals`.

Reply:
`ReferenceEquals` is a static method so I don't think we can actually use `this`. We could use `object` if we wanted though Visual Studio was telling me to remove that.